src/core/Dropdown/Dropdown/Dropdown.md

```diff
@@ -1,238 +1,378 @@
+The `<Dropdown>` component is used to select a value from a list of options. It should be used when the amount of options is manageable in size, i.e the user does not need filtering to find the correct option.
```

Comment:
```suggestion
The `<Dropdown>` component is used to select a value from a list of options. It should be used when the amount of options is manageable in size, i.e the user does not need filtering to find the correct option. The component is rendered as a combination of a `button` element and an ARIA listbox.
```

Reply:
This. Plus a dot after the e in "i.e." 😁

Reply:
Done!